### PR TITLE
GPU options for the Perfetto trace config dialog.

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

**Comment by pau-baiget** [comment]
Can you include a snapshot of the UI in this PR? Thank you!

**Comment by stellama0208** [comment]
> Can you include a snapshot of the UI in this PR? Thank you!

Will be like this:
https://photos.app.goo.gl/LCPDTivodNDQuo8U7

File: gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java [hunk 1]
```diff
@@ -173,31 +221,94 @@ protected void okPressed() {
...
+        }
+
+        if (gpuCaps.getGpuCounterDescriptor().getSpecsCount() > 0) {
+          gpuCounters = createCheckbox(
+              gpuGroup, "Counters", settings.perfettoGpuCounters, e -> updateGpu());
+          Composite counterGroup = withLayoutData(
+              createComposite(gpuGroup, withMargin(new GridLayout(3, false), 5, 0)),
+              withIndents(new GridData(), GROUP_INDENT, 0));
+          gpuCountersLabels = new Label[3];
+          gpuCountersLabels[1] = createLabel(counterGroup, "Poll Rate:");
```
Comment (stellama0208) [inline gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java]: nit: maybe can change the index sequence to natural order 0, 1, 2?

**Comment by lpy** [comment]
\o/

**Pull request merged**